tools/accuracy_checker/configs/opencv/alexnet.yml

```diff
@@ -0,0 +1,55 @@
+models:
+  - name: AlexNet
+
+    launchers:
+      - framework: opencv
+        device: CPU
+        tags:
+          - FP32
```

Comment:
now you are able do not set device in config, it allows decreasing config file size

Reply:
could I have you show example